### DISCUSSION TITLE
Italian Update to .net8 oqtane 5.1.2

### DISCUSSION THREAD

**Discussion by acasiraro** [oqtane/oqtane.translations#55] [General]
I am not yet familiar with GitHub but I would like to contribute with the Italian translation.

So far I have followed these steps:
1. I created a copy of the project [https://github.com/oqtane/oqtane.translations] on my Github account and forked it.
2. I cloned the repository to my local PC.
3. I duplicated the .resx files from the oqtane.client project taken from [https://github.com/oqtane/oqtane.framework].
4. Once I created the .it-IT.resx files, I translated all the files.
5. I committed the changes to my repository and synchronized them to Github.

Now the files are located at [https://github.com/acasiraro/oqtane.translations].

Let me know if any step is unclear or incorrect.

If all steps are correct, I would like to be able to update the translation project and know how to do it.
If possible, please do it for me at the moment.

Thank you

**Comment by leigh-pointer** [comment]
@acasiraro thank you very much for that translations. One thing i have noticed is that you are missing translations for the Oqtane Server project.

On a side note, I am busy creating some nice templates and features for translations.

**Comment by acasiraro** [comment]
thank you @leigh-pointer 
I have add Oqtane.server too.

I hadn't added it earlier because I hadn't found it in the previous Italian translation.

**Comment by leigh-pointer** [comment]
@acasiraro  The translations have been merged.  Building the package is easy enough but it also needs to be deployed to Oqtane Marktplace.  This needs to be done by the maintainer because the package id is linked to the Oqtane marketplace account.  Maybe I should thinks about thais a little longer, I'm sure I will find away.

**Reply by leigh-pointer** [reply]
@acasiraro  Translation is released
